docs: Update customer portal docs/screenshots (#9604)
zitadel/zitadel: docs/docs/guides/start/quickstart.mdx

![Login](/img/guides/quickstart/v3_8.png)

### 2. Complete onboarding questions

To begin, we'll ask you a few questions.
These will help us understand your needs and personalize your ZITADEL experience, ensuring you get the most out of it.

![Onboarding Questions](/img/guides/quickstart/onboarding_questions.png)

### 3. Create your first instance

As a user of the ZITADEL Cloud Customer Portal, you now can create multiple instances to suit your specific needs.
This includes instances for development, production, or user acceptance testing, as well as instances for different clients or applications.

> For example, you might create an instance for each product in a B2C scenario, or an instance for each tenant or customer in a B2B scenario.

Reviewer: ```suggestion
```

Reviewer: I would leave this out here. Could create confusion. I guess the default for such scenarios would be to use organizations. Some cases might require instances, but I guess that's overload here and could irritate.

The possibilities are endless. You can create your first instance for free.

> 1. Let’s create an instance. Click on “Create Instance”.

Reviewer: ```suggestion
1. Create an instance.
by clicking on “Create Instance”.
```

3. After creating your first instance, you're ready to configure user authentication. To access the management console, click the 'Sign in to your instance' button.

![Sign In Instance](/img/guides/quickstart/sign_in_instance.png)

6. To log in to your instance, provide the username and password you set in the instance creation, and click “next”.

7. Skip the 2-factor authentication for now by clicking “skip”.

8. And there you go! You now have access to your instance.

![Instance Details](/img/guides/quickstart/instance_dashboard.png)

### 4. Create your first project and app

Because applications are always associated with a project, we'll guide you through setting up both in the following steps.

1. Click the "Create Application" button or the Framework/Language of your choice on the right side

2. Insert “Project1” (or any name of your choice) as the project’s name and select the framework/language of your choice in our case "React" and click the “Continue” button. (If you can't find your framework/languages select "Other")

![Project Name and Framework/Language](/img/guides/quickstart/create_project_react.png)

> 3. You will now see an overview of the configuration for the framework you have selected

Reviewer: ```suggestion
3. An overview of the configuration for the framework you have selected is shown
```

![Project Configuration Overview - React App](/img/guides/quickstart/project_config_overview_react.png)

> For some clarification what the configuration does:

Reviewer: ```suggestion
What the configuration does:
```

- We recommend using the [Authentication Code Flow](/docs/apis/openidoauth/grant-types#authorization-code) with [Proof Key for Code Exchange (PKCE)](/docs/apis/openidoauth/grant-types#authorization-code) for secure OIDC configuration. This setup includes specific App, Grant, and Response Types, as well as an Authentication Method. More about the different app types can be found [here](/docs/guides/integrate/login/oidc/oauth-recommended-flows#different-client-profiles).
- The redirect URI is where the ZITADEL authorization server redirects the user after they have been authenticated. The "Redirect URI" is preconfigured to [http://localhost:3000/callback](http://localhost:3000/callback) and the “Post Logout URI” to [http://localhost:3000/](http://localhost:3000/) for the React example

> 4. Now you will see the detail configuration of your app.

Reviewer: ```suggestion
4. The detailed configuration of your app will be shown
```

![Project Name and Framework/Language](/img/guides/quickstart/app_detail_overview_react.png)

### 5. Add users (optional)

> [Skip optional steps](quickstart/#7-create-an-application-in-your-project) and jump directly to the Obtain ClientId and OIDC endpoints.

Reviewer: ```suggestion
[Skip optional steps](quickstart/#7-create-an-application-in-your-project) and jump directly to the obtain ClientId and OIDC endpoints.
```
